JobRecruiting/JobRecruiting/src/main/java/com/recruit/jobrecruiting/servlet/jobposting/AddJobPost.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.recruit.jobrecruiting.servlet.jobposting;
+
+import com.recruit.jobrecruiting.ejb.JobPostBean;
+import com.recruit.jobrecruiting.ejb.SkillBean;
+import com.recruit.jobrecruiting.entity.Department;
+import com.recruit.jobrecruiting.entity.Status;
+import com.recruit.jobrecruiting.entity.Type;
+import com.recruit.jobrecruiting.validators.JobPostValidator;
+import java.io.IOException;
+import java.util.HashMap;
+import javax.inject.Inject;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ *
+ * @author DENISA
+ */
+@WebServlet(name = "AddJobPost", urlPatterns = {"/JobPost/Create"})
+public class AddJobPost extends HttpServlet {
+
+    @Inject
+    private JobPostBean jobPostBean;
+
+    @Inject
+    private SkillBean skillBean;
+
+    /**
+     * Handles the HTTP <code>GET</code> method.
+     *
+     * @param request servlet request
+     * @param response servlet response
+     * @throws ServletException if a servlet-specific error occurs
+     * @throws IOException if an I/O error occurs
+     */
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+
+        request.setAttribute("departments", Department.values());
+        request.setAttribute("skills", skillBean.getAllSkills());
```

Comment:
Iti functioneaza linia asta? Eu primesc NullPointerException si am skills in BD.

Reply:
Scuze, se pare ca am avut eu o problema cu serverul. Functioneaza 👍 Good Job!